rocky-raft/src/rocky/raft/dto/ConfigEntry.java

```diff
@@ -0,0 +1,40 @@
+package rocky.raft.dto;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class ConfigEntry implements Serializable {
+    private List<Address> currentConfig;
+    private List<Address> newConfig;
```

Comment:
But how will we differentiate between a joint configuration log entry vs new configuration log entry?